src/apps/clients/notifications.clj

```diff
@@ -57,11 +57,11 @@
    cemerick.url.
 
    Example usage:
-       (str (interapps-url (url (config/interapps-base)) job-info))
+       (str (interapps-url (url (config/interapps-base)) username external-id))
      Returns:
        https://abb9730df.cyverse.run"
-  [{host :host :as base} {user-id :user_id analysis-id :uuid}]
-  (assoc base :host (str "a" (-> (str user-id analysis-id) sha256 (subs 0 8)) "." host)))
+  [{host :host :as base} username external-id]
+  (assoc base :host (str "a" (-> (str username external-id) sha256 (subs 0 8)) "." host)))
```

Comment:
I made a slight change to this function because the job information available in the apps service is slightly different from the job information available to condor-launcher. It's confusing, but unfortunately, it's easier to change how we get the information than it is to change the available job information at this time.